src/main/java/org/lwes/ArrayEvent.java

```diff
@@ -160,25 +158,29 @@ public void clear(String key) {
 
     @Override
     public void setEventName(String name) {
-        checkShortStringLength(name, encoding, MAX_EVENT_NAME_SIZE);
+        checkShortStringLength(name, MAX_EVENT_NAME_SIZE);
         final String oldName = getEventName();
```

Comment:
why does this even allow an event name to be changed?  would be easier if the name were set in the constructor and not changeable.  oh well